jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BottomNavigationTests.java

```diff
@@ -47,19 +47,19 @@ public void hideOnScrollTest() {
     @Test
     public void shiftTest() {
         bottomNavigationShift.get(1).has().text("Video");
```

Comment:
more clear to work through text or/and class then order number. Such step can be excess

Reply:
sure need to go with text instead of index

Reply:
But I think we kept it here for better usability in case someone will really need to use access by index.